pkg/controllers/management/auth/project_cluster/project_handler.go

```diff
@@ -155,6 +155,11 @@ func (l *projectLifecycle) reconcileProjectCreatorRTB(obj runtime.Object) (runti
 			return obj, fmt.Errorf("expected project, got %T", obj)
 		}
 
+		// If we specify no creator owner RBAC, exit
+		if _, ok := project.Annotations[NoCreatorRBACAnnotation]; ok {
```

Comment:
since this is inside the `DoUntilTrue`, will it set the `CreatorMadeOwner` condition to true? If that is the case, would it be possible to move this outside of the `CreatorMadeOwner.DoUntilTrue` ? so we don't set the condition `CreatorMadeOwner` as we don't make the creator owner if the `NoCreatorRBACAnnotation` is set.

Reply:
That's a good point. I think we should also move the check that the object passed is a project. If it's not a project it's never going to be true.